odoo/tools/misc.py

```diff
@@ -124,16 +136,17 @@ def exec_pg_environ():
 def exec_pg_command(name, *args):
     prog = find_pg_tool(name)
     env = exec_pg_environ()
-    with open(os.devnull) as dn:
-        args2 = (prog,) + args
-        rc = subprocess.call(args2, env=env, stdout=dn, stderr=subprocess.STDOUT)
-        if rc:
-            raise Exception('Postgres subprocess %s error %s' % (args2, rc))
+    subprocess.run((prog, *args), env=env, stdout=subprocess.DEVNULL, stderr=subprocess.STDOUT, check=True)
 
 def exec_pg_command_pipe(name, *args):
```

Comment:
Maybe also deprecate the two `exec_pg_command`?

They don't do much, useful, they're only used 3 times in all, and one of these uses doesn't even seem correct:
```python
        if odoo.tools.exec_pg_command(pg_cmd, *pg_args):
            raise Exception("Couldn't restore database")
```
that condition and specialised error messages haven't done anything useful since ec9a543014a91be17398ba6e443578fe37c3a885